data/schemas/edges/coding_variants_phenotypes.VAMPAdapter.json

```diff
@@ -1,138 +1,144 @@
+          "description": "The biological context of the coding variant to phenotype edge",
+          "items": {
+            "type": "string"
+          },
+          "example": "HAP-1"
+        },
+        "biosample_term": {
+          "type": "string",
+          "description": "The biosample term of the coding variant to phenotype edge",
+          "items": {
+            "type": "string"
+          },
+          "example": "ontology_terms/EFO_0002067"
+        },
+        "method": {
+          "enum": [
+            "VAMP-seq"
+          ]
+        },
+        "label": {
+          "enum": [
+            "mutational effect"
+          ]
+        },
+        "class": {
+          "enum": [
+            "observed data"
+          ]
+        },
+        "score": {
+          "type": [
+            "number",
+            "null"
+          ],
+          "description": "Score for the coding variant to phenotype edge"
+        },
+        "standard_error": {
+          "type": [
+            "number",
+            "null"
+          ],
+          "description": "Standard error for the coding variant to phenotype edge"
+        },
+        "rep1_score": {
+          "type": [
+            "number",
+            "null"
+          ],
+          "description": "Rep1 score for the coding variant to phenotype edge"
+        },
+        "rep2_score": {
+          "type": [
+            "number",
+            "null"
+          ],
+          "description": "Rep2 score for the coding variant to phenotype edge"
+        },
+        "rep3_score": {
+          "type": [
+            "number",
+            "null"
+          ],
+          "description": "Rep3 score for the coding variant to phenotype edge"
+        }
       },
-      "example": [
-        "HAP-1"
-      ]
-    },
-    "method": {
-      "type": "string",
-      "description": "Method for the coding variant to phenotype edge",
-      "example": "VAMP-seq"
-    },
-    "biological_context": {
-      "type": [
-        "string",
-        "null"
-      ],
-      "description": "Biological context for the coding variant to phenotype edge",
-      "example": "ontology_terms/EFO_0001184"
-    },
-    "score": {
-      "type": [
-        "number",
-        "null"
-      ],
-      "description": "Score for the coding variant to phenotype edge"
-    },
-    "standard_error": {
-      "type": [
-        "number",
-        "null"
-      ],
-      "description": "Standard error for the coding variant to phenotype edge"
-    },
-    "rep1_score": {
-      "type": [
-        "number",
-        "null"
-      ],
-      "description": "Rep1 score for the coding variant to phenotype edge"
-    },
-    "rep2_score": {
-      "type": [
-        "number",
-        "null"
-      ],
-      "description": "Rep2 score for the coding variant to phenotype edge"
-    },
-    "rep3_score": {
-      "type": [
-        "number",
-        "null"
+      "required": [
+        "_key",
+        "_from",
```

Comment:
remove _from, _to, source, source_url, name, method since they are already required in edges.base.json. Should add biosample_term as required.